src/main/java/com/onarandombox/MultiverseCore/commands/CreateCommand.java

```diff
@@ -34,12 +33,12 @@ public class CreateCommand extends MultiverseCommand {
     public CreateCommand(@NotNull MultiverseCore plugin) {
         super(plugin);
 
-        this.plugin.getCommandManager().getFlagsManager().registerFlagGroup(FlagGroup.builder("mvcreate")
-                .add(MVValueFlag.builder("--seed", String.class)
+        this.flagsManager.registerFlagGroup(CommandFlagGroup.builder("mvcreate")
```

Comment:
Why not just `registerFlagGroup`? Also, can we drop the `this`?

Reply:
hmm yea

uhh its just a habit that i put this for all class field, so i didnt confuse myself when i was new to java